src/libs/actions/Report.ts

```diff
@@ -2438,27 +2441,30 @@ function navigateToConciergeChat(shouldDismissModal = false, checkIfCurrentPageA
     }
 }
 
-function buildNewReportOptimisticData(policy: OnyxEntry<Policy>, reportID: string, reportActionID: string, reportName: string, creatorPersonalDetails: PersonalDetails) {
+function buildNewReportOptimisticData(policy: OnyxEntry<Policy>, reportID: string, reportActionID: string, creatorPersonalDetails: PersonalDetails, reportPreviewReportActionID: string) {
     const {accountID, login} = creatorPersonalDetails;
     const parentReport = getPolicyExpenseChat(accountID, policy?.id);
     const {stateNum, statusNum} = getExpenseReportStateAndStatus(policy);
     const timeOfCreation = DateUtils.getDBTime();
-    const reportPreviewActionID = rand64();
+    const titleReportField = getTitleReportField(getReportFieldsByPolicyID(policy?.id) ?? {});
 
     const optimisticDataValue: OptimisticNewReport = {
         reportID,
         policyID: policy?.id,
         type: CONST.REPORT.TYPE.EXPENSE,
         ownerAccountID: accountID,
-        reportName,
         stateNum,
         statusNum,
         total: 0,
         nonReimbursableTotal: 0,
         participants: {},
         lastVisibleActionCreated: timeOfCreation,
+        pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
```

Comment:
@sumo-slonik I think we should be adding the `parentReportID` and `parentReportActionID` to the report here to avoid the layout shift when creating a report. We should also optimistically add the report next steps key

https://github.com/user-attachments/assets/d5312921-674d-49fe-9f68-38739230b703

Reply:
I'm already getting on with fixing it

Reply:
managed to make it work as you suggested:

https://github.com/user-attachments/assets/00c7a39a-13a0-4230-86b5-e8dc590cd53a

Reply:
Nice!